membership-common/src/main/scala/com/gu/monitoring/SafeLogger.scala

```diff
@@ -45,11 +49,16 @@ class SafeLoggerImpl(logger: Logger) {
     logger.warn(logMessage, throwable)
   }
 
-  def error(logMessage: LogMessage): Unit = {
+  def errorNoPrefix(logMessage: LogMessage): Unit = {
     logger.error(logMessage.withPersonalData)
     logger.error(SafeLogger.sanitizedLogMessage, logMessage.withoutPersonalData)
   }
 
+  def error(logMessage: LogMessage)(implicit logPrefix: LogPrefix): Unit = {
+    logger.error(logPrefix.message + ": " + logMessage.withPersonalData)
+    logger.error(SafeLogger.sanitizedLogMessage, logMessage.withoutPersonalData)
+  }
```

Comment:
this is the new method for error.  I didn't add it to the sentry message as sentry is supposed to aggregate similar messages.
TODO in the next PR, add the equivalent for info, warn etc..